.github/workflows/pr.yaml

```diff
@@ -1,25 +1,38 @@
 name: PR
 on: pull_request
 jobs:
-  build:
-    name: Build
+  end2end:
+    name: End To End
```

Comment:
We can separate out building go code and pushing docker images from the helm deployments eventually. There's not really much code to do this with though, so the renaming may have been a preoptimization.